### ISSUE TITLE
Xdagger

### ISSUE THREAD

**Issue by maiquecoutinho** [RainbowMiner/RainbowMiner#2369]
Can you give the correct configuration for XDAG? please

**Comment by maiquecoutinho** [comment]
[debug_2023-05-23.zip](https://github.com/RainbowMiner/RainbowMiner/files/11547762/debug_2023-05-23.zip)

Only WARNING: No pools available: check your configuration

I try pool.xdag.org:13656 and stratum.xdag.org:23656

**Comment by RainbowMiner** [comment]
I don't think we have an XDAG miner integrated yet - I will add one as soon as possible.
